src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -66,6 +69,55 @@ public async Task SendAsync_SpecialCharacterHeader_Success()
             });
         }
 
+        [Fact]
+        public async Task GetAsync_MissingExpires_ReturnNull()
+        {
+             await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
+             {
+                using (var client = CreateHttpClient())
+                {
+                    HttpResponseMessage response = await client.GetAsync(uri);
+                    Assert.Null(response.Content.Headers.Expires);
+                }
+            },
+            async server =>
+            {
+                await server.HandleRequestAsync(HttpStatusCode.OK);
+            });
+        }
+
+        [Theory]
+        [InlineData("Thu, 01 Dec 1994 16:00:00 GMT")]
+        [InlineData("-1")]
+        [InlineData("0")]
+        public async Task SendAsync_Expires_Success(string value)
+        {
+            await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
+            {
+                using (var client = CreateHttpClient())
+                {
+                    var message = new HttpRequestMessage(HttpMethod.Get, uri);
+                    HttpResponseMessage response = await client.SendAsync(message);
+                    Assert.NotNull(response.Content.Headers.Expires);
```

Comment:
There should an Assert.Equals() here.  What value do we expect to see from the strongly typed property 'Expires' ?

Reply:
When invalid value is added via TryAddWithoutValidation() and read, back it has same value as stored e.g. 0, -1 or what ever. When such value is received in responser and parsed, it will have value of DateTime.MinValue when read from strongly typed Content.Expires.